### ISSUE TITLE
Feedback: Unit 1 Lab 5: Bits, Games and Violence (1-intro-loops)

### ISSUE THREAD

**Issue by cycomachead** [bjc-edc/bjc-r#105]
#### [View the Page here](http://bjc.edc.org/bjc-r/cur/programming/1-introduction/5-intro-bits-games/big-questions.html?topic=nyc_bjc%2F1-intro-loops.topic&course=bjc4nyc_2015-2016.html&novideo&noassignment)

---

A page with a lot of readings is hard to follow. All the links open in the same window so there's no easy way to go back. You can get distracted. Also doing all the reading in one page makes the pacing of the lab a bit weird -- one page usually conveys a similar amount of work but this psge iis long in comparison.

**Comment by maryfries** [comment]
Inputted as a comment on the HTML page
